src/pages/ChatPage/components/ChatInput.tsx

```
@@ -0,0 +1,87 @@
+      className="border-border relative flex flex-col items-end gap-1 border-t px-2 py-4 md:flex-row md:items-center"
+    >
+      <div className="relative flex w-full flex-1 items-center gap-1">
+        <input
+          {...register("message")}
+          placeholder={t("chat.sendMessage", "Send message")}
+          className="border-border flex-1 rounded-lg border bg-muted-10 px-4 py-2 text-body focus:outline-none focus:ring-2 focus:ring-primary-20"
+        />
+        <span
+          className="absolute end-2 cursor-pointer text-neutral hover:text-primary"
+          onClick={() => setOpen(!open)}
+        >
```

Biome (2.3.13)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)